src/runtime/recipe/util.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Recipe} from './recipe.js';
```

Comment:
The name seems too generic, but recipe-util is already taken...
This is not the best way to determine whether `recipe` is a `RequireSection`, but importing `RequireSection` causes a cycle (tried to move it to its own file, but then it's a cycle between that and recipe). 
Should I add a TODO to find a nicer way?

Reply:
Sure, a TODO sounds fine. Maybe we can just set a boolean on the recipe when we create it? Just dreaming without looking at code.

Reply:
added a todo (waiting for tests to pass and merging...)